test/TimeService.Data.UnitTests/WorkTimeMonthLimitRepositoryTests.cs

```diff
@@ -0,0 +1,278 @@
+      };
+
+      await _repository.CreateRangeAsync(dbWorkTimeMonthLimits);
+
+      Assert.AreEqual(3, await _dbContext.WorkTimeMonthLimits.CountAsync());
+    }
+
+    [Test]
+    public async Task CreateRangeReturnNullAsync()
+    {
+      await _repository.CreateRangeAsync(null);
+
+      Assert.AreEqual(0, await _dbContext.WorkTimeMonthLimits.CountAsync());
+    }
+
+    [Test]
+    public async Task CreateRangeReturnNullByOneElementAsync()
+    {
+      List<DbWorkTimeMonthLimit> dbWorkTimeMonthLimits = new()
+      {
+        _firstWorkTimeMonthLimit, null, _thirdWorkTimeMonthLimit
+      };
+
+      await _repository.CreateRangeAsync(dbWorkTimeMonthLimits);
+
+      Assert.AreEqual(0, await _dbContext.WorkTimeMonthLimits.CountAsync());
+    }
+
+    [Test]
+    public async Task EditSuccessfullyAsync()
+    {
+      List<DbWorkTimeMonthLimit> list = new() { _editableWorkTimeMonthLimit };
+      IQueryable<DbWorkTimeMonthLimit> queryable = list.AsQueryable();
+      Mock<DbSet<DbWorkTimeMonthLimit>> dbSet = new Mock<DbSet<DbWorkTimeMonthLimit>>();
+
+      dbSet.As<IQueryable<DbWorkTimeMonthLimit>>()
+        .Setup(x => x.Provider)
+        .Returns(queryable.Provider);
+      dbSet.As<IQueryable<DbWorkTimeMonthLimit>>()
+        .Setup(m => m.Expression)
+        .Returns(queryable.Expression);
+      dbSet
+        .As<IQueryable<DbWorkTimeMonthLimit>>()
+        .Setup(m => m.ElementType)
+        .Returns(queryable.ElementType);
+      dbSet
+        .As<IQueryable<DbWorkTimeMonthLimit>>()
+        .Setup(m => m.GetEnumerator())
+        .Returns(() => queryable.GetEnumerator());
+
+      _mocker
+        .Setup<IHttpContextAccessor, IDictionary<object, object>>(x => x.HttpContext.Items)
+        .Returns(_items);
+      _mocker
+        .Setup<IDataProvider, IEnumerable<DbWorkTimeMonthLimit>>(x => x.WorkTimeMonthLimits)
+        .Returns(dbSet.Object);
+
+      WorkTimeMonthLimitRepository mockedRepository = _mocker.CreateInstance<WorkTimeMonthLimitRepository>();
+
+      string value = "Edited Holidays";
+      JsonPatchDocument<DbWorkTimeMonthLimit> request = new()
+      {
+        Operations = { new() { op = "replace", path = "/Holidays", value = value } }
+      };
+
+      Assert.IsTrue(await mockedRepository.EditAsync(_editableWorkTimeMonthLimit.Id, request));
+      Assert.AreEqual(value, _editableWorkTimeMonthLimit.Holidays);
+    }
+
+    [Test]
+    public async Task EditReturnFalseAsync()
+    {
+      await _dbContext.AddAsync(_firstWorkTimeMonthLimit);
+      await _dbContext.SaveChangesAsync();
+
+      Assert.IsFalse(await _repository.EditAsync(_secondWorkTimeMonthLimit.Id, null));
+    }
+
+    [Test]
+    public async Task EditReturnFalseByRequestAsync()
+    {
+      await _dbContext.AddAsync(_firstWorkTimeMonthLimit);
+      await _dbContext.SaveChangesAsync();
+
+      Assert.IsFalse(await _repository.EditAsync(_firstWorkTimeMonthLimit.Id, null));
```

Comment:
Why you don not use SerializerAssert? It is our class in UnitTestKernel.

Reply:
I noticed it doesn't work correctly and return true when it's actually not